test/cases/040_packages/032_bcc/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# SUMMARY: Check that the bcc package works
+# LABELS:
```

Comment:
can you make this:
```
# LABEL: skip
```
for now. That will skip the test and I can enable it once the we have the BCC package build.